src/opl.c

```diff
@@ -1655,8 +1655,12 @@ static void init(void)
     setDefaults();
 
     padInit(0);
-    configInit(NULL);
-
+    int padStatus = 0;
+    while (!padStatus)
+        padStatus = startPads();
+    readPads();
+    if (!getKeyPressed(KEY_START))
+        configInit(NULL);//only load settings if emergency key is not being pressed
```

Comment:
Comment should be indented by space bedore and after //

This is good commit but it requires at least some note in Readme, also it is not obvious that exactly START key should be hold, so START button should be mentioned in commit description.

Reply:
> Comment should be indented by space bedore and after //
> 
> This is good commit but it requires at least some note in Readme, also it is not obvious that exactly START key should be hold, so START button should be mentioned in commit description.

Mention it on the squash and merge commit

Reply:
But what about nentioning in Readme.md